package.json

```diff
@@ -459,7 +459,7 @@
               "type": "PowerShell",
               "request": "launch",
               "script": "^\"\\${file}\"",
-              "cwd": "^\"\\${file}\""
+              "cwd": "^\"\\${workspaceFolder}\""
```

Comment:
@rkeithhill Do you know why this was `file` before? It seemed quite wrong to me, as the thing is asking for a directory, and our other task (below) was given `workspaceFolder` too.

Reply:
I found my answer at long last...https://github.com/PowerShell/vscode-powershell/blob/3dc11c1fd671bd814dce75bffdff5f737aa817fe/src/features/DebugSession.ts#L304-L306